R/drive_update.R

```diff
@@ -1,19 +1,23 @@
-#' Update a file on Drive with new content.
+#' Update an existing Drive file
+#'
+#' @seealso Wraps the
+#' [drive.files.update](https://developers.google.com/drive/v3/reference/files/update)
+#' endpoint. In particular, does [media upload](https://developers.google.com/drive/v3/web/manage-uploads).
 #'
 #' @param file Character, path to the local file to upload.
 #' @template path
-#' @param ... Query parameters to pass along to the API query.
+#' @param ... Parameters to pass along to the API query. NOT IMPLEMENTED YET.
```

Comment:
Some of these will go in the body, once we truly support metadata updates (https://github.com/tidyverse/googledrive/issues/148).